reference/storage/file-signUrl.md

```diff
@@ -0,0 +1,63 @@
+Create a signed url for access to a file
```

Comment:
Change this file name to file-sign-url.md or file-signurl.md

Reply:
done. I think we can skip snippets for now as we haven't redone them for this new resource centric approach. Can revisit with broader language support I think.